pulse2percept/tests/test_api.py

```diff
@@ -22,7 +22,7 @@ def test_Simulation_pulse2percept():
         sim.pulse2percept(pt)
 
     pt = p2p.stimuli.BiphasicPulse("cathodicfirst", 0.1, 0.005 / 1000)
-    sim.pulse2percept(pt)
+    sim.pulse2percept(pt, layers=['GCL', 'invalid'])

```

Comment:
Currently it produces a warning and ignores the layer. But I can upgrade it to an error.